src/action/list_directory.rs

```diff
@@ -356,15 +347,14 @@ impl super::Response for Response {
             st_flags_linux: self.flags_linux,
             symlink:
             self.symlink
-                .map_or(None,
-                        |symlink| Some(symlink.to_string_lossy().to_string())),
+                .map(|symlink| symlink.to_string_lossy().to_string()),
             registry_type: None,
             resident: None,
             pathspec: Some(rrg_proto::PathSpec {
-                // represents CaseLiteral path option (other options are not
-                // supported)
+                // Represents `CaseLiteral` path option (other options are not
+                // supported).
                 path_options: Some(Options::CaseLiteral as i32),
-                // represents OS path type (other types are not supported)
+                // Represents `OS` path type (other types are not supported).
```

Comment:
"OS" is actually not an identifier, it's just an abbreviation of "operating system", so should not be backticked.

Reply:
It can be treated as abbreviation or one of enum options in protobuf(same as `CaseLiteral`), I suppose. Nevertheless, fixed.